README.md

```diff
@@ -2,10 +2,43 @@ Intro
 =====
 This repository contains the backend logic for the dfusion exchange based on the specification, see [here](github.com/gnosis/dex-research)
 
-
-Instructions
+Architecture
 ============
+<p align="center">
+<img src="documentation/architecture.png" alt="dex-services architecture" width="500">
```

Comment:
Is this the correct location and filename? I see `dƒusion architecture.png` as the filename in the root directory.

Reply:
It's because the PR is currently rebased on an old version of master. After the merge it will look fine.